ts/components/ui/Markdown/index.tsx

```diff
@@ -298,9 +298,7 @@ class Markdown extends React.PureComponent<Props, State> {
     const containerStyle: ViewStyle = {
       height: htmlBodyHeight + (extraBodyHeight || 0)
     };
-
-    const isLoading =
-      html === undefined || (html !== "" && htmlBodyHeight === 0);
+    const isLoading = html === undefined;
```

Comment:
It would be awesome to see a unit test covering this specific bug to ensure won't represent itself in the future.